Dockerfile

```diff
@@ -20,6 +20,9 @@ LABEL com.centurylinklabs.watchtower.enable="true"
 ## add package tarball
 COPY *.tar.gz .
 
+## add package DESCRIPTION file
```

Comment:
The source code of the R package (qmongr) does not exist in this container, only the installed version of the package. Copying the DESCRIPTION file into the container will not have the effect intended here...

To avoid updating the [qmongr-base-r](https://github.com/SKDE-Felles/qmongr-base-r/blob/master/Dockerfile) image repeatedly during development on might (temporarily) add a similar update when building this image, _e.g._

``` RUN R -e "install.packages(c('cli','yaml'))"```

Reply:
Strange... This at least fixed the travis build.